src/pages/ServicePage.js

```diff
@@ -37,6 +37,13 @@ class ServicePage extends React.Component {
         // FIXME: display error on failure to connect
     }
 
+    /**
+     * Pull out the ID from the slug
+     */
+    id(): number {
+        return parseInt(this.props.params.id);
```

Comment:
Use a regex to remove the id from the slug instead of relying on parseInt semantics.